index.js

```diff
@@ -6,47 +6,58 @@ function subtract(a,b){
   return a-b
 }
 
-function verifyInput(number1,number2){
-  if((typeof number1 !== 'number')||(typeof number2 !== 'number')){
+function verifyInput(num1,num2){
+  if ((Number.isInteger ( num1 )) && (Number.isInteger ( num2 ))){
+    if (num1 > 0 && num2 > 0){
+      return true
+    } else {
+      return false
+    }
+  } else {
     return false
   }
 }
 
-function isDivideByTwo(num){
-  return (num%2===0)
-}
+function getLcd(num1, num2){
+  let mini_num = Math.min(num1, num2);
```

Comment:
in javascript we tend to use capital letter in const such as:

miniNum

Reply:
Thanks for the advice.
I will fix it.